core/templates/components/forms/schema-based-editors/schema-based-unicode-editor.directive.html

```diff
@@ -1,25 +1,28 @@
 <span>
   <textarea ng-if="!$ctrl.getCodingMode() && $ctrl.getRows()"
+            class="form-control"
             ng-model="$ctrl.localValue"
-            rows="<[$ctrl.getRows()]>"
-            placeholder="<[$ctrl.getPlaceholder()]>"
             ng-disabled="$ctrl.isDisabled()"
-            class="form-control" focus-on="<[$ctrl.labelForFocusTarget()]>"
             ng-blur="$ctrl.onInputBlur()"
-            ng-focus="$ctrl.onInputFocus()">
+            ng-focus="$ctrl.onInputFocus()"
+            focus-on="<[$ctrl.labelForFocusTarget()]>"
+            placeholder="<[$ctrl.getPlaceholder()]>"
+            apply-validation validators="$ctrl.validators()"
+            ng-keypress="$ctrl.onKeypress($event)"
+            rows="<[$ctrl.getRows()]>">
```

Comment:
I reordered the attributes here to match the attributes below in the `input` tag. More importantly, I added `apply-validation validators="$ctrl.validators()"` -- before, we did not perform any validations on `textarea` objects.